cpp/src/join/conditional_join.cu

```diff
@@ -45,6 +142,14 @@ conditional_join(table_view const& left,
                  rmm::cuda_stream_view stream,
                  rmm::mr::device_memory_resource* mr)
 {
+  // Idk enough about libcudf to know if this code is called directly. I left it here just in case,
+  // I don't think it should be needed since this function isn't meant to be called directly anyway.
+  // Will remove on request
```

Comment:
You can assume that code in `detail` namespaces is not called by public consumers, it is only called within cudf. If that's the case, perhaps it is safe to remove?

Reply:
Sounds good, will remove.